doc/release-process.md

```diff
@@ -183,9 +183,9 @@ Non-codesigners: wait for Windows/macOS detached signatures:
 - Once the Windows/macOS builds each have 3 matching signatures, they will be signed with their respective release keys.
 - Detached signatures will then be committed to the [bitcoin-detached-sigs](https://github.com/bitcoin-core/bitcoin-detached-sigs) repository, which can be combined with the unsigned apps to create signed binaries.
 
-Create (and optionally verify) the codesigned outputs:
+Create (and optionally verify) the codesigned build outputs:
```

Comment:
As "optionally verify" is mentioned here, I would suggest to add the https://github.com/bitcoin/bitcoin/blob/ba7d150cf49dcdfac7080c6adbf419e7b4b9b016/contrib/guix/README.md#verifying-build-output-attestations link, or do not mention "optionally verify" at all.

Reply:
Done